gradle.properties

```diff
@@ -4,3 +4,4 @@ kotlin.parallel.tasks.in.project=true
 systemProp.org.gradle.kotlin.dsl.caching.buildcache=true
 systemProp.gradle.publish.skip.namespace.check=true
 systemProp.http.keepAlive=false
+systemProp.org.gradle.unsafe.vfs.partial-invalidation=true
```

Comment:
Shall we add some comment here as to what this does and how long we are planning to keep it here?

Reply:
Good idea.

I am also wondering if we should print a message when partial invalidation is enabled, so that we somehow see that the system property is picked up. WDYT?

Reply:
I added a comment.

Reply:
Good idea about printing a message. `Partial VFS invalidation is an incubating feature`-like?

We should print one for `VFS retention`, too.